{"name": "test_case", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 7", "trace": "self = <test.demo1.test_pytest-allure2.Test_case object at 0x0441AAD0>\n\n    @allure.story(\"定义场景2\")\n    def test_case(self):\n        with allure.step(\"定义步骤2\"):\n            C =case().add(a,b)\n        with allure.step(\"断言2\"):\n>           assert C == 7\nE           assert 3 == 7\n\ntest_pytest-allure2.py:13: AssertionError"}, "steps": [{"name": "定义步骤2", "status": "passed", "start": 1585048164211, "stop": 1585048164211}, {"name": "断言2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 7\n", "trace": "  File \"d:\\python_work\\test\\demo1\\test_pytest-allure2.py\", line 13, in test_case\n    assert C == 7\n"}, "start": 1585048164211, "stop": 1585048164211}], "attachments": [{"name": "stdout", "source": "a2688390-4bfb-4cea-99da-37762da4bc56-attachment.txt", "type": "text/plain"}], "start": 1585048164211, "stop": 1585048164212, "uuid": "4494640b-29fc-41b1-9a09-9fb0949ad4ab", "historyId": "70b3cc93fe42ead085744f5cde99956a", "testCaseId": "0688394cee7576038edfe5b52c8cfaa8", "fullName": "test_pytest-allure2.Test_case#test_case", "labels": [{"name": "feature", "value": "定义功能2"}, {"name": "story", "value": "定义场景2"}, {"name": "suite", "value": "test_pytest-allure2"}, {"name": "subSuite", "value": "Test_case"}, {"name": "host", "value": "law"}, {"name": "thread", "value": "19644-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest-allure2"}]}